packages/elements/src/containers/API.tsx

```diff
@@ -51,14 +51,20 @@ export interface CommonAPIProps extends RoutingProps {
    * @default "sidebar"
    */
   layout?: 'sidebar' | 'stacked';
+  logo?: {
+    url: string;
+    backgroundColor?: string;
+    altText?: string;
+    href?: string;
+  };
```

Comment:
Are we looking to add logo support as a prop, too? cc @philsturgeon

Reply:
> 2 missing pieces:
> 
> 1. Poor stacked layout has no logo support. Is that intentional?
> 2. No tests. It'd be especially important to test an `x-logo` spec to see if transformation & display work as a whole.
> 
> If these two are handled, it should be good to go, I see no problems with the code written at all.:)

@marcelltoth 
1. No design requirements for the StackedLayout, you're right. @philsturgeon - where do you think we should put it?
2. Spec transformation is already being tested in `http-spec` (if what you mean is tranlating users document to `httpService`). Unless you're talking about testing data flow, but I wouldn't personally test the outcome (display) directly, since we agreed on not testing the implementation (which I would need to do in order to check the logo). We can do visual testing of course, but not in this PR probably

Reply:
Yes, we want users to be able to provide it both as a prop and from the API document, as described in https://github.com/stoplightio/elements/issues/706